<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>911</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>304</width>
        <height>503</height>
       </rect>
      </property>
      <attribute name="label">
       <string>QListWidget操作</string>
      </attribute>
      <widget class="QToolButton" name="tBtnListInit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>tBtnListInit</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tBtnListClear">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>tBtnListClear</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tBtnListInsert">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>tBtnListInsert</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tBtnListAppend">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>tBtnListAppend</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tBtnListDelete">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>tBtnListDelete</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>304</width>
        <height>503</height>
       </rect>
      </property>
      <attribute name="label">
       <string>QTreeWidget操作</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>304</width>
        <height>503</height>
       </rect>
      </property>
      <attribute name="label">
       <string>QTableWidget操作</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="listWidgetTab">
      <attribute name="title">
       <string>QListWidget</string>
      </attribute>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>671</width>
         <height>461</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>firstItem</string>
        </property>
        <property name="checkState">
         <enum>Checked</enum>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Images/item.png</normaloff>Images/item.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SecondItem2</string>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Images/item.png</normaloff>Images/item.png</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>当前变化项</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>20</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="text">
        <string>可编辑</string>
       </property>
      </widget>
      <widget class="QToolButton" name="selectAll">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>selectAll</string>
       </property>
      </widget>
      <widget class="QToolButton" name="selectNon">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>selectNon</string>
       </property>
      </widget>
      <widget class="QToolButton" name="selcetInvert">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>selcetInvert</string>
       </property>
      </widget>
      <widget class="QToolButton" name="selectItem">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="text">
        <string>selectItem</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="treeWidgetTab">
      <attribute name="title">
       <string>QTreeWidget</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tableWidgetTab">
      <attribute name="title">
       <string>QTableWidget</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="toolTip">
    <string>这是状态栏</string>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolTip">
    <string>这是工具栏</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actListInit">
   <property name="text">
    <string>初始化列表</string>
   </property>
  </action>
  <action name="actListClear">
   <property name="text">
    <string>清空列表</string>
   </property>
   <property name="toolTip">
    <string>清空列表</string>
   </property>
  </action>
  <action name="actListInsert">
   <property name="text">
    <string>插入项</string>
   </property>
   <property name="toolTip">
    <string>插入项</string>
   </property>
  </action>
  <action name="actListAppend">
   <property name="text">
    <string>添加项</string>
   </property>
  </action>
  <action name="actListDelete">
   <property name="text">
    <string>删除当前项</string>
   </property>
   <property name="toolTip">
    <string>删除当前项</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
